api/AnimationPlaybackEvent.json

```diff
@@ -4,18 +4,60 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationPlaybackEvent",
         "support": {
+          "webview_android": {
+            "version_added": false
+          },
           "chrome": {
             "version_added": false
           },
+          "chrome_android": {
+            "version_added": false
```

Comment:
All Chrome versions for all interface members should be `true`. This was renamed from `AnimationPlayerEvent` in Chrome 55.

Reply:
So, this would be a candidate for `alternative_name` and two support ranges then.

```json
"chrome_android": [
  {
    "version_added": "55"
  },
  { 
    "version_added": true,
    "alternative_name": "AnimationPlayerEvent"
  }
]
```

What do you think @jpmedley ?

Reply:
should that change go into all three chrome versions?

Reply:
Yes.